README.md

```diff
@@ -593,4 +649,4 @@ required to build the library.
 ### Windows
 
 * Users must manually link `shlwapi.lib`. Failure to do so may result
-in resolved symbols.
\ No newline at end of file
+in resolved symbols.
```

Comment:
Yes, I guess so. Anyway, apparently this diff here is my editor's fault - it adds a newline at the last character if there isn't one.